code/modules/jobs/jobs.dm

```diff
@@ -1,87 +1,83 @@
 GLOBAL_LIST_INIT(command_positions, list(
-	"Captain",
-	"Head of Personnel",
-	"Head of Security",
-	"Chief Engineer",
-	"Research Director",
-	"Chief Medical Officer"))
+	"Captain", "Station Commander", "Facility Overseer",
+	"Head of Personnel", "Chief of Staff", "Head of Internal Affairs",
+	"Head of Security", "Security Commander", "Security Chief",
+	"Chief Engineer", "Head of Engineering", "Engineering Director",
+	"Research Director", "Chief Science Officer", "Head of Research",
+	"Chief Medical Officer", "Medical Director", "Head of Medical"))
 
 
 GLOBAL_LIST_INIT(engineering_positions, list(
-	"Chief Engineer",
-	"Station Engineer",
-	// yogs start - Yog jobs
-	"Atmospheric Technician",
-	"Signal Technician"))
-	// yogs end
+	"Chief Engineer", "Head of Engineering", "Engineering Director",
+	"Station Engineer", "Engine Technician", "Solar Engineer", "Project Engineer", "Junior Engineer", "Construction Specialist",
+	"Atmospheric Technician", "Breach Fixer", "Habitation Technician", "Fire Supression Technician", "Atmospherics Trainee", "Environmental Maintainer",
+	"Signal Technician", "NTSL Programmer", "Comms Tech", "Station IT Support"
+	))
 
 
 GLOBAL_LIST_INIT(medical_positions, list(
-	"Chief Medical Officer",
-	"Medical Doctor",
-	"Geneticist",
-	"Virologist",
-	// yogs start - Yog jobs
-	"Chemist",
-	"Paramedic",
-	"Psychiatrist",
-	"Mining Medic"))
-	// yogs end
+	"Chief Medical Officer", "Medical Director", "Head of Medical",
+	"Medical Doctor", "Physician", "Surgeon", "Nurse", "Medical Resident", "Attending Physician", "Chief Surgeon", "Attending Physician", "Chief Surgeon", "Medical Subdirector", "General Practitioner",
```

Comment:
Double Chief Surgeon, remove Medical Subdirector as potentially too confusing in line with Ident's review